packages/shared/lib/participation/participation.ts

```diff
@@ -91,7 +91,19 @@ export const canParticipate = (eventState: ParticipationEventState): boolean =>
  *
  * @returns {boolean}
  */
-export const canAccountParticipate = (account: WalletAccount): boolean => account?.rawIotaBalance >= DUST_THRESHOLD
+export const canAccountParticipate = (account: WalletAccount): AccountParticipationAbility => {
```

Comment:
Should we use another function name? May be slightly confusing as it reads like it will return a `boolean`.

Reply:
Updated.